semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/SymbolInformationOps.scala

```diff
@@ -226,13 +226,22 @@ trait SymbolInformationOps { self: SemanticdbOps =>
       }
     }
 
+    def overridenSymbols: List[String] = {
+      if (gsym.overrideChain.size > 1) {
```

Comment:
This method can be replaced with `gsym.allOverriddenSymbols`, which returns `Nil` if this method doesn't override symbols.

Note that this is an expensive operation that's not cached, so it's good to avoid duplicate `overrideChain`.

Reply:
Changed it base don suggestions, thanks!